apps/finance/app/src/components/Transfers.js

```diff
@@ -128,7 +123,7 @@ class Transfers extends React.PureComponent {
     const { selectedDateRange } = this.state
     const start = format(selectedDateRange.start, 'yyyy-MM-dd')
     const end = format(selectedDateRange.end, 'yyyy-MM-dd')
-    return `Finance_(${start}_to_${end}).csv`
+    return `transfers_${start}_to_${end}.csv`
```

Comment:
Could we use the Finance app's proxy address here, like `0x1a0adf8909(Finance)_transfers_...`